frontend/src/app/[locale]/opportunity/[id]/page.tsx

```diff
@@ -40,7 +40,7 @@ export async function generateMetadata({
   let title = `${t("OpportunityListing.page_title")}`;
   try {
     const { data: opportunityData } = await getOpportunityDetails(id);
-    title = `${t("OpportunityListing.page_title")} - ${opportunityData.opportunity_title}`;
+    title = `${t("OpportunityListing.page_title")} - ${opportunityData.opportunity_title || ""}`;
```

Comment:
Not a blocker since the scope of this is pr is typing, but is this translating the title twice? Could this be: 
```suggestion
    title = `${title} - ${opportunityData.opportunity_title || ""}`;
```

Reply:
discussed with Freddy, and this would probably be a more dry implementation but we'll leave as is for now:
```
let opportunityData;
  try {
    const { data } = await getOpportunityDetails(id);
    opportunityData = data;
  } catch (error) {
    console.error("Failed to render page title due to API error", error);
    if (parseErrorStatus(error as ApiRequestError) === 404) {
      return notFound();
    }
  }
  const pageTitle = t("OpportunityListing.page_title");
  const title = opportunityData?.opportunity_title
    ? `${pageTitle} - ${opportunityData.opportunity_title}`
    : pageTitle;
```